packages/components/src/frame-provider/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import { CacheProvider } from '@emotion/core';
+import createCache from '@emotion/cache';
+import memoize from 'memize';
+
+const memoizedCreateCacheWithContainer = memoize( ( container ) => {
+	return createCache( { container } );
+} );
```

Comment:
What's the difference with `useMemo`?

Reply:
The only difference  I can think of is that useMemo is tied to the component but I think it's fine here.